src/item.tsx

```diff
@@ -58,9 +51,9 @@ const getInterpolatedStyles = (
     return null;
   }
 
-  const style: InterpolatedStyles = { transform: [{ perspective: 1000 }] };
+  const style: any = { transform: [{ perspective: 1000 }] };
```

Comment:
After updating the dependencies there was 3 `tsc` errors, which gets dismissed with this `any`. (I did try to fix it otherwise before but couldn't - feel free if you feel like, but I was honestly thinking it's not worth it.)

Reply:
And for the process I was waiting for the review to hear your opinions, I was thinking creating a new release indeed? 
As we're targeting `8fit/carousel-animation#v0.2.1` it won't be using this new version before we explicitly change it, so we are safe that nothing will change on prod, and we'll be able to test with the new version?

I don't think documentation or changelog are of any value, if we want to invest time here it probably make sense to not use this forked package anymore.

Reply:
> After updating the dependencies there was 3 `tsc` errors, which gets dismissed with this `any`. (I did try to fix it otherwise before but couldn't - feel free if you feel like, but I was honestly thinking it's not worth it.)

Okay, fair enough. As I'm not sure how much development effort will be invested into the 8fit app in the future, it's probably okay to not spend a lot of time on figuring out these issues.

> And for the process I was waiting for the review to hear your opinions, I was thinking creating a new release indeed?

I have zero knowledge about this repository 😅 
I just saw that there are indeed GH releases: https://github.com/8fit/carousel-animation/releases
So, yeah, we should keep those then, I think.

> As we're targeting `8fit/carousel-animation#v0.2.1` it won't be using this new version before we explicitly change it, so we are safe that nothing will change on prod, and we'll be able to test with the new version?

I'm actually wondering where our app gets this package from, because I just searched npm and there's no `8fit/carousel-animation` package when you search for "8fit": https://www.npmjs.com/search?q=8fit

🤔 

Apparently it's using the GitHub URL syntax: https://docs.npmjs.com/cli/v7/configuring-npm/package-json#github-urls
🤯 I didn't know that was a thing!
So, then nothing would need to be published anywhere. We would just need to create a GH release, in my understanding.
And then, yes, as the GitHub URL includes a version tag builds using the previous version (`v0.2.1`) should still build with it and only builds using the version number that we set for the next version would include this changeset.

Then, if we're not publishing this package anywhere we're also pretty free in defining the next version number. We should probably make it `v0.3.0`.

> I don't think documentation or changelog are of any value, if we want to invest time here it probably make sense to not use this forked package anymore.

I thought that there might already be a changelog in place and then it would be good to stick to it. But if that's not the case (and I just looked around and didn't see any changelog) then definitely we shouldn't introduce one.

> it probably make sense to not use this forked package anymore.

I don't think this package is a fork?! The initial commit looks like it's been developed from scratch: https://github.com/8fit/carousel-animation/commit/b649a0a7b7475e86ad2ac14a41baf415d42f04a6

Reply:
Yeah, looks like this was originally developed by Mo (a former 8fit dev): https://github.com/8fit/carousel-animation/pull/1#pullrequestreview-277713793